policyengine_us/parameters/gov/states/ri/tax/income/credits/cdcc/rate.yaml

```
@@ -0,0 +1,13 @@
+description: Rhode Island matches the Federal Credit for child and dependent care expenses by this rate.
+values:
+  2021-01-01: 0.25
+metadata:
+  unit: /1
+  label: Rhode Island CDCC match percent
+  reference:
+    - title: 2022 Form RI-1040 Resident Individual Income Tax Return - page 3
+      href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022_1040WE_w_0.pdf
+    - title: 2021 Form RI-1040 Resident Individual Income Tax Return - page 3
+      href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-01/2021_1040we_w.pdf
+    - title: R.I. Gen. Laws § 44-30-2.6
```

I don't understand why it restarts from N to A, but this is the second (F)
```suggestion
    - title: R.I. Gen. Laws § 44-30-2.6(F)(I)(g)
```